### PR TITLE
Container option

### PR DESCRIPTION
**Pull request by johnsenpeder**
PR based on code changes/idea by @alexalexalex-s https://github.com/zenorocha/clipboard.js/issues/155#issuecomment-273124130

Fixes issues with using clipboard in bootstrap modals etc.

### CODE REVIEW

**Comment by iest** [comment]
@zenorocha any progress on this? This would also solve an issue when using with `react-modal2`.

Happy to help make improvements to get it moving.

File: readme.md [hunk 1]
```diff
@@ -151,6 +151,15 @@ new Clipboard('.btn', {
...
 ```
 
+For use in bootstrap modals you'll want to set the modal element as the `container` value.
+See [Issue #155 (comment)](https://github.com/zenorocha/clipboard.js/issues/155#issuecomment-273124130)
+
+```js
+new Clipboard('.btn', {
+    container: document.getElementById('#modal')
+});
+```
```
Comment (iest) [inline readme.md]: This issue isn't exclusive to bootstrap — you get the same issue with `react-modal2` or any other library that changes the focus.

Might be worth clarifying this?

Comment (johnsenpeder) [inline readme.md]: @iest updated the text

**Comment by johnsenpeder** [comment]
Any chance this might get merged anytime soon? :)

**Comment by adrianbj** [comment]
I would like to see this also. I need to refer to `document.documentElement`, rather than `document.body`. Currently I am hacking clipboard.min.js to do this,  but I assume this PR would let me do this more gracefully?

**Comment by seanmhanson** [comment]
+1 for getting this resolved and merged

**Comment by ryantheleach** [comment]
Thanks for merging this I had run into an issue with jQuery Mobile popups. I suspect this library with this PR will fix this.

If only I knew about the issues I would see when I decided to use SO to come up with a copy script rather then use your library. Then maybe I wouldn't have spent 5 hours debugging. well, Still probably would have, but would have found the container option faster.

https://stackoverflow.com/questions/44303217/why-does-copying-stop-working-from-the-context-menu

**Pull request merged**